core/domain/services/graphql/mutators/DatetimeUpdate.php

```diff
@@ -83,13 +87,24 @@ public static function mutateAndGetPayload(EEM_Datetime $model, Datetime $type)
             }
 
             // Update the entity
-            $result = $entity->save($args);
-
-            if (empty($result)) {
-                throw new UserError(esc_html__('The object failed to update but no error was provided', 'event_espresso'));
+            try {
+                $entity->save($args);
+            } catch (Exception $exception) {
+                new ExceptionStackTraceDisplay(
+                    new RuntimeException(
+                        sprintf(
+                            esc_html__(
+                                'The Datetime failed to update because of the following error(s):%1$s%2$s',
+                                'event_espresso'
+                            ),
+                            '<br/>',
```

Comment:
GraphQL clients usually treat those messages as escaped text, HTML is no rendered.